src/applications/gi-sandbox/sass/partials/_gi-search-controls.scss

```diff
@@ -71,12 +71,12 @@
 }
 .update-results-form {
   margin-top: -2em;
-  border-left: solid #f1f1f1;
-  border-right: solid #f1f1f1;
+  border-left: solid $color-gray-lightest;
+  border-right: solid $color-gray-lightest;
```

Comment:
`$color-gray-lightest` is `#f1f1f1`.

https://design.va.gov/design/color-palette

![image](https://user-images.githubusercontent.com/2008881/119417663-333d0a80-bcab-11eb-91a4-c9db0fa214f6.png)